PowerValidatedSolutions.psm1

```diff
@@ -24359,7 +24362,7 @@ Function Connect-vRSLCMUpgradeIso {
         - Connects the ISO to the VMware Aria Suite Lifecycle appliance
 
         .EXAMPLE
-        Connect-vRSLCMUpgradeIso -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -contentLibrary Operations -libraryItem "VMware-Aria-Suite-Lifecycle-Appliance-8.14.0.4-22630472-updaterepo"
+        Connect-vRSLCMUpgradeIso -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -contentLibrary Operations -libraryItem "VMware-Aria-Suite-Lifecycle-Appliance-x.y.z-xxxxxxxx-updaterepo"
```

Comment:
Shoudnt this reflect the actual build?

Reply:
I made it generic so that we don't need to change that level of details for each release but happy to do that. Lemme know if you have it!